### PR TITLE
Adds Atom feed View to both Django and Flask

### PR DESCRIPTION
**Pull request by b-m-f**
Using werzeugs Atom Feed generator.

## QA

Implement the blog module in your project in `requirements.txt` by adding `git+https://github.com/b-m-f/blog-extension@add_atom_feed#egg=canonicalwebteam.blog`.

Consume the feed with `from canonicalwebteam.blog.django.views import feed`.

The parameters are `def feed(request, tags_exclude=[], tags=[], title="", subtitle=""):`.

Point a url to this view and confirm that a correct XML is returned.

### CODE REVIEW

**Comment by pmahnke** [comment]
@b-m-f 	not that I got anwhere, however, I am getting this message

```
Double requirement given: canonicalwebteam.blog from git+https://github.com/b-m-f/blog-extension@add_atom_feed#egg=canonicalwebteam.blog (from -r requirements.txt (line 19)) (already in canonicalwebteam.blog==2.0.14 (from -r requirements.txt (line 3)), name='canonicalwebteam.blog')

```

**Comment by b-m-f** [comment]
@pmahnke oh yeah, need to remove any previous entries for the blog module.

File: canonicalwebteam/blog/common_view_logic.py [hunk 1]
```diff
@@ -241,12 +243,33 @@ def get_archives(self, page=1, group="", month="", year="", category=""):
 
         return context
 
-    def get_feed(self, uri):
-        feed = api.get_feed(self.tag_name)
-        right_urls = logic.change_url(feed, uri.replace("/feed", ""))
-        context = right_urls.replace("Ubuntu Blog", self.blog_title)
+    def get_feed(
+        self, uri, tags_exclude=[], tags=[], title="Ubuntu Blog", subtitle=""
```
Comment (jpmartinspt) [inline canonicalwebteam/blog/common_view_logic.py]: Can we default title to something more generic or make it required? "Ubuntu Blog" is quite specific.

File: canonicalwebteam/blog/wordpress_api.py [hunk 2]
```diff
@@ -43,11 +43,7 @@ def build_url(endpoint, params={}):
 
     query = urlencode(clean_params)
```
Comment (jpmartinspt) [inline canonicalwebteam/blog/wordpress_api.py]: What do you thing about making this line `urlencode({**clean_params, '_embed': "true"})`, and then `return f"{API_URL}/{endpoint}?{query} `

File: canonicalwebteam/blog/common_view_logic.py (hunk 1 above)
Comment (b-m-f) [reply canonicalwebteam/blog/common_view_logic.py]: Changed it to "Blog"

File: canonicalwebteam/blog/wordpress_api.py (hunk 2 above)
Comment (b-m-f) [reply canonicalwebteam/blog/wordpress_api.py]: Nice one. Updated it. As discussed and test going from `&_embed` to `&_embed=true` does not seem to influence the functionality

**Pull request merged**